aim/sdk/objects/image.py

```diff
@@ -101,29 +106,52 @@ def json(self):
             'height': self.height,
         }
 
-    def _from_pil_image(self, pil_image: PILImage.Image):
+    def _from_pil_image(self, pil_image: PILImage.Image, compress=False):
         assert isinstance(pil_image, PILImage.Image)
-        image_bytes_p = BytesIO()
-        pil_image.save(image_bytes_p, format='png')
+        img_container = BytesIO()
+
+        if compress:
+            if pil_image.mode == 'RGBA' or isinstance(pil_image.info.get('transparency'), bytes):
```

Comment:
@devfox-se please comment why we need this step

Reply:
@alberttorosyan already have a comment underneath that line:)